Stardrop/Converters/EnumToBoolConveter.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Avalonia.Data.Converters;
+using System;
+using System.Globalization;
+
+namespace Stardrop.Converters
+{
+    public class EnumToBoolConveter : IValueConverter
```

Comment:
hm, on second thought, a better name for this might just be `EnumEqualsConverter`.

Reply:
I agree, `EnumEqualsConverter` is a bit more clear.